src/main/java/com/palettee/chat/controller/dto/response/ChatResponse.java

```diff
@@ -1,47 +1,76 @@
 package com.palettee.chat.controller.dto.response;
 
+import com.palettee.chat.controller.dto.request.ChatRequest;
 import com.palettee.chat.domain.Chat;
 import com.palettee.chat.domain.ChatImage;
+import com.palettee.chat_room.domain.ChatRoom;
+import com.palettee.global.redis.utils.TypeConverter;
+import com.palettee.user.domain.User;
 import lombok.*;
 
 import java.time.LocalDateTime;
 import java.util.List;
+import java.util.UUID;
 
 @Getter
+@Setter
+@Builder
 @AllArgsConstructor
-@NoArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class ChatResponse {
     private Long chatRoomId;
-    private Long chatId;
     private String email;
     private String profileImg;
     private String content;
-    private List<ChatImgUrlResponse> imgUrls;
-    private LocalDateTime timestamp;
+    private List<ChatImgUrl> imgUrls;
+    private String sendAt;
 
-    public static ChatResponse toResponse(Long chatRoomId, Chat chat, List<ChatImage> chatImages) {
-        if(chatImages == null) {
+    public static ChatResponse toResponse(Long chatRoomId, User user, ChatRequest chatRequest) {
+        if(chatRequest.imgUrls() == null && chatRequest.imgUrls().isEmpty()) {
```

Comment:
조건이 잘못된 것 같습니다.

```suggestion
        if(chatRequest.imgUrls() == null || chatRequest.imgUrls().isEmpty()) {
```

아니면 삼항연산자 이용해서 아래처럼 만들수도 있을 것 같습니다.

```java
List<ChatImgUrl> imageUrl = chatRequest.imgUrls();

return new ChatResponse(
        chatRoomId,
        user.getEmail(),
        user.getImageUrl(),
        chatRequest.content(),
        imageUrl != null && !imageUrl.isEmpty() ?
                imageUrl : null,
        TypeConverter.LocalDateTimeToString(LocalDateTime.now())
);
```

Reply:
삼항 연산자 좋은 방법인 것 같아요!! 코드를 확 줄여주네요 👍